staging/src/k8s.io/apiserver/pkg/storage/testing/utils.go

```diff
@@ -232,10 +232,8 @@ func (p *PrefixTransformer) TransformToStorage(ctx context.Context, data []byte,
 	return data, p.err
 }
 
-func (p *PrefixTransformer) GetReads() uint64 {
-	return atomic.LoadUint64(&p.reads)
-}
-
-func (p *PrefixTransformer) ResetReads() {
+func (p *PrefixTransformer) GetReadsAndReset() uint64 {
+	result := atomic.LoadUint64(&p.reads)
```

Comment:
Shouldn't this be a `SwapUnit64`?

Reply:
done - good catch
[although probably doesn't matter in any of the existing tests]

[Also fixed in another place in the third commit]